unique_toolkit/unique_toolkit/language_model/schemas.py

```diff
@@ -193,33 +193,32 @@ class LanguageModelStreamResponse(BaseModel):
 
 
 class LanguageModelTokenLimits(BaseModel):
-    token_limit: Optional[int] = None
     token_limit_input: Optional[int] = None
     token_limit_output: Optional[int] = None
+    token_limit: Optional[int] = None
```

Comment:
Looks like the order changed. Hope it isn't used anywhere without specifying the param name.

Reply:
Initializing pydantic models is not possible afaik

![Screenshot 2024-12-12 at 09 41 47](https://github.com/user-attachments/assets/1204f1ba-cd38-4d0f-aed1-528ca3c56b13)